### DISCUSSION TITLE
Welcome to Stash_Helper Discussions!

### DISCUSSION THREAD

**Discussion by philpw99** [philpw99/Stash_Helper#31] [Announcements]
Just post your questions here.

**Comment by PalmerRobbie** [comment]
Getting error trying to install and launch this. Need some assistance. Put screenshots in an Issue on here

**Comment by PalmerRobbie** [comment]
I've got the program running after the web driver fix was released last week, I think. I have a menu of options from Stash Helper running in my System Tray, but nothing does anything. I'm not sure at all how to use this application, do you have any tutorial videos or something???

I tried Ctrl+Right Mouse button while on a Stash page, nothing comes up but my regular Windows context menu, not the menu as shown on this forum for "Add to Playlist" and other features.

**Comment by PalmerRobbie** [comment]
And are you supposed to not have your web browser open before launching Stash Helper? It tells me it will close my web browser (Edge), then I have to wait almost three minutes for my web browser to reopen itself. At that point I get "Error Getting data from the stash server. Returned code: 401". I'd guess because I have to authenticate to the database, but there's no option anywhere in Stash Helper to add authentication credentials to the Stash database?

![image](https://github.com/user-attachments/assets/1c0bbd6c-8da9-4399-b64f-88f886f42e5b)

Am I missing something?

**Comment by PalmerRobbie** [comment]
I played around with this some more, and I think I'm trying to use features from the System Tray application of Stash Helper, but it comes back with that "401 error" over and over when trying to retrieve something from the Stash Server. If someone could validate that is because my database requires authentication, or that error means something else, I'd appreciate it. I'd like to finally try to use this Stash Add-On

**Comment by PalmerRobbie** [comment]
> Just post your questions here.

Is anybody responsible for this thread?

**Comment by Luststrolch** [comment]
In Case this topic is not dead:

Has anybody figured out how to add multiple scenes to a playlist?
I can only add ALL scenes Stash currently has - which is far too much! - and filters seem to be ignored.

**Reply by philpw99** [reply]
Just figure out the reason it won't add multiple scenes to the play list.
Now you can add thousands of scenes to a playlist with one click, or filter out the scenes then add them to the playlist.
https://github.com/philpw99/Stash_Helper/releases/tag/v2.5.1

**Comment by philpw99** [comment]
Wow! Seems you are probably the only one who actually want to use this program. 
I was quite actively involved in Stash during 2023 and 2024, but then since no one is interested I went on to do other things.
This program has issues here and there, and I never tested it with authentication.
Actually to bypass the Stash built-in authentication, it's not hard. I am going to work on it.

Just tested with user/pass. It works. The helper will automatically detect the login screen and get the ApiKey once you login.
You need to generate one API key in "Settings -> Security", otherwise it will be always 401 error.

Also you can set the Stash Helper in "Remote" mode to test first, even the IP is 127.0.0.1.
When Stash Helper is in "Local" mode, it will try to read the "config.yml" file and it might cause a problem.

When you exit the program, be sure to close the robot-control Firefox as well, Stash Helper should close them for you, but sometimes it will miss extra browser instances, then there will be errors like data error and 401.

**Comment by Luststrolch** [comment]
THX! 👍

Am So., 25. Jan. 2026 um 23:47 Uhr schrieb Philip Wang <
***@***.***>:
